charts/orchestrator/README.md

```diff
@@ -75,6 +75,18 @@ The following table lists the configurable parameters of the Orchestrator chart
 | `tekton.enabled` | whether to create the Tekton pipeline resources | `false` |
 | `argocd.enabled` | whether to install the ArgoCD plugin and create the orchestrator AppProject | `false` |
 | `argocd.namespace` | Defines the namespace where the orchestrator's instance of ArgoCD is deployed. The value is captured when running setup.sh script and stored as a label in the selected namespace. User can override the value by populating this field. Defaults to `orchestrator-gitops` in the setup.sh script. | `""` |
+| `rhsso.enabled` | Whether to create the Red Hat SSO login information | `true` |
+| `rhsso.deploy` | If we need to deploy the RedHat SSO operator (For development pourposes) | `true` |
+| `rhsso.metadataURL` | If it's defined will use this, if not will build based on the deploy details. | `""` |
+| `rhsso.baseURL` | If it's defined will use this, if not will build based on the deploy details. | `""` |
```

Comment:
can you elaborate on what these variables are for? the comment matches for both variables and I am unsure if they are related in any way.

Reply:
updated the comment.